src/mappings/veNft/veNftDeployer.ts

```diff
@@ -0,0 +1,38 @@
+import { DeployVeNft } from "../../../generated/JBVeNftDeployer/JBVeNftDeployer";
+import {
+  Project,
+  DeployedVeNftEvent,
+  VeNftContract,
+} from "../../../generated/schema";
+import { idForProject, idForProjectTx, idForVeNftContract } from "../../utils";
+import { JBVeNft } from "../../../generated/templates";
+
+export function handleDeployVeNft(event: DeployVeNft): void {
+  let cv = "2";
+  let projectId = idForProject(event.params.projectId, cv);
+  let project = Project.load(projectId);
+  if (!project) return;
+  let deployedVeNftEvent = new DeployedVeNftEvent(
+    idForProjectTx(event.params.projectId, cv, event)
+  );
+  if (deployedVeNftEvent) {
+    deployedVeNftEvent.project = project.id;
+    deployedVeNftEvent.projectId = project.projectId;
+    deployedVeNftEvent.timestamp = event.block.timestamp.toI32();
+    deployedVeNftEvent.txHash = event.transaction.hash;
+    deployedVeNftEvent.save();
```

Comment:
let's also update the `saveNewProjectEvent` function to create a `ProjectEvent` for this. I see you already added `ProjectEvent.deployedVeNftEvent` 🙏

for context, the `ProjectEvent` is like a union entity that allows different project events to be queried in a single list (i.e. in the project dashboard activity view)

Reply:
got it will do 🍻